crates/fluvio-protocol/src/link/error_code.rs

```diff
@@ -57,6 +57,12 @@ pub enum ErrorCode {
     #[fluvio(tag = 71)]
     #[error("Offset {offset} is evicted. The next available is {next_available}")]
     OffsetEvicted { offset: i64, next_available: i64 },
+    #[fluvio(tag = 72)]
+    #[error("Storage is full")]
```

Comment:
Should be something like PartitionFull/PartitionShortCircuited

Reply:
fixed